wnfs-wasm/tests/public.spec.ts

```diff
@@ -178,9 +178,9 @@ test.describe("PublicDirectory", () => {
         store
       );
 
-      var { result: imagesContent } = await rootDir.ls(["images"], store);
+      const imagesContent = await rootDir.ls(["images"], store);
 
-      var { result: picturesContent } = await rootDir.ls(["pictures"], store);
+      const picturesContent = await rootDir.ls(["pictures"], store);
 
```

Comment:
Cool :+1: I'm onboard with this change for read-only functions on the wasm-wnfs side :)